Registration/RegisterWithAzure.psm1

```diff
@@ -346,7 +346,7 @@ function Set-AzsRegistration{
         [switch] $MarketplaceSyndicationEnabled = $true,
 
         [Parameter(Mandatory = $false, ParameterSetName = "Register")]
-        [switch] $UsageReportingEnabled = @{'Capacity'=$true; 
+        [switch] $UsageReportingEnabled = @{'Capacity'=$false; 
                                             'PayAsYouUse'=$true; 
```

Comment:
Is this a breaking change?

Reply:
this is not a breaking change, just disabling it for capacity billing model by default.